src/main/java/io/appform/dropwizard/discovery/bundle/resolvers/DefaultNodeInfoResolver.java

```diff
@@ -0,0 +1,21 @@
+package io.appform.dropwizard.discovery.bundle.resolvers;
+
+import io.appform.dropwizard.discovery.bundle.ServiceDiscoveryConfiguration;
+import io.appform.ranger.common.server.ShardInfo;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class DefaultNodeInfoResolver implements
+  NodeInfoResolver {
+
+  private static final String FARM_ID = "FARM_ID";
+
+  @Override
+  public ShardInfo node(ServiceDiscoveryConfiguration configuration) {
+    return ShardInfo.builder()
+      .environment(configuration.getEnvironment())
+      .region(System.getenv(FARM_ID))
```

Comment:
Can we add a log to print the actual value of FARM_ID here? It is possible that FARM_ID may not be set at all. It would be good to warn the user that in the absence of $FARM_ID, the library resorts to using a default value which can be derived based on the hostname or could be set to "NA" or "UNKNOWN".

Reply:
Debug log, probably yes. Setting defaults is not a good idea in here. The defaults may be specific to the client (org) that is using it, I feel.